content/content-strategy/audit-content/cards.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: cards/cards
+cards:
+  - headline: Prove the value
+    text: One of the first questions you will need to answer when pitching a project to audit content, is what the return on investment will be.
+    link: /content-strategy/audit-content/prove-value/
+  - headline: Plan your audit
+    text: To plan your content audit, focus on the extent each website or application is still fulfilling its purpose.
+    link: /content-strategy/audit-content/plan%20your%20audit/
```

Comment:
This is a broken link with the '%20' in it. SHould be `/plan-audit`

Reply:
Trisha fixed link